### ISSUE TITLE
Cache not saving on Windows

### ISSUE THREAD

**Issue by amyreese** [hynek/setup-cached-uv#15]
Seems to be working just fine on other OS/py combinations, including Windows 3.12 and Linux/Mac 3.13.

During post job, the following output is generated:

```
Post job cleanup.
Post job cleanup.
$ uv cache prune --ci
Post job cleanup.
Warning: Path Validation Error: Path(s) specified in the action for caching do(es) not exist, hence no cache is being saved.
```

gha job: https://github.com/omnilib/aioitertools/actions/runs/10660169548/job/29543776557?pr=193

gha workflow: https://github.com/omnilib/aioitertools/pull/193/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f

**Comment by hynek** [comment]
this is #12, no? they wrote not restoring but it's not restoring because it's not saving :D

**Comment by hynek** [comment]
BTW I checked – it's not saving on your other combinations either which blessedly makes it a bit simpler.

**Comment by amyreese** [comment]
ok, I swear I'd looked and the 3.12 windows job had saved, so I thought it was something specific to 3.13, but now that I look again I must have clicked on the wrong jobs when trying to diagnose this. 🤪

Unlike #12 though, I'm not specifying any particular shell or anything else "weird" beyond installing to the "system" so that I don't have to manage venvs in my CI jobs. Is there a workaround in the other issue that I'm missing?

**Comment by hynek** [comment]
No, I think it’s my fault and will have to figure out what Windows directories are off-limits if accessed from a different action. :( IOW: I suspect I can access certain dirs in my own CI, but not if the action is used in a different project.

this is gonna be fun to figure out

**Comment by hynek** [comment]
looks like Astral has published their own action and I don't think there's a point in competing with them so maybe check them out and see if it works over there: https://github.com/astral-sh/setup-uv

**Comment by amyreese** [comment]
> looks like Astral has published their own action and I don't think there's a point in competing with them so maybe check them out and see if it works over there: https://github.com/astral-sh/setup-uv

It looks like Astral's action does cache/restore correctly on windows, using a non-standard temp directory:

```
Set UV_CACHE_DIR to D:\a\_temp\setup-uv-cache
```

https://github.com/omnilib/ufmt/actions/runs/10766175508/job/29851507811?pr=239

**Comment by hynek** [comment]
lol i just saw it too

**Comment by hynek** [comment]
fixed by #16
